### ISSUE TITLE
The mapping file used for analyses should include the taxonomic summaries

### ISSUE THREAD

**Issue by ElDeveloper** [biocore/qiita#1390]
The QIIME mapping file that's used to produce the visuals in Qiita should include information about the taxonomic profiles i.e. we should be calling summarize_taxa.py on the table an mapping file before starting any of the processing.

Side note: To make this nicer for the users, we should perhaps trim the taxonomic name to something more readable, meaning that instead of naming the column `Root;k__Bacteria;p__Actinobacteria;c__Actinobacteria`, we can probably use `c__Actinobacteria`.

**Comment by antgonza** [comment]
This has been superseded by how Qiime2 works; thus, closing.
